lib/spack/spack/graph.py

```diff
@@ -80,12 +80,14 @@ def topological_sort(spec, **kwargs):
 
     """
     reverse = kwargs.get('reverse', False)
+    # XXX(deptype): iterate over a certain kind of dependency. Maybe color
+    #               edges based on the type of dependency?
```

Comment:
This sounds cool.  But don't worry too much about `spack graph` for now -- we can always make it nicer later.